source-firestore/.snapshots/TestDeletions-two

```diff
@@ -1,9 +1,4 @@
 # ================================
-# Collection "flow_source_tests/*/docs": 2 Documents
-# ================================
-{"_meta":{"path":"projects/project-id-123456/databases/(default)/documents/flow_source_tests/TestDeletions/docs/3","mtime":"<TIMESTAMP>","delete":true}}
```

Comment:
Do you know why this snapshot changed?

Reply:
Not fully, no. Best I can tell, the tests were just broken. Which makes sense, given that they aren't run as part of CI, so it'd be easy for them to get outdated. But the test in this case clearly deletes those two documents from the source prior to verifying the snapshot. And the behavior of them being missing from the snapshot seems consistent with the `TestDeletions-one` snapshot, which also doesn't include the deleted documents.